sample/kotlin/src/main/AndroidManifest.xml

```diff
@@ -37,8 +37,7 @@
         <activity android:name=".ViewPagerActivity"/>
 
         <service
-            android:name=".service.LogsForegroundService"
-            android:process=":logsService"/>
```

Comment:
why it cannot be in the dedicated process now?

Reply:
For the sake of this test, a different process means a new Zygote instance therefore a new SDK instance which will work independently from the Main process (meaning now actual views in foreground)